### ISSUE TITLE
An error has occurred while applying security settings. Users is not a valid user or group.

### ISSUE THREAD

**Issue by MightTower** [GPUOpen-Tools/CodeXL#2]
During installation of CodeXL (in the "Start services" step) the error "An error has occurred while applying security settings. Users is not a valid user or group." pops up. Retry does not change anything. The setup is canceled. I have a WIndows 7 64 bit (german) Version.
For me adding a new Group (in lusrmgr.msc) with name "User" and my account as a user in this group solves the problem. I think this maybe a localization problem since in German the group is called "Benutzer", but I am not sure.

**Comment by klausklaus** [comment]
Confirming the problem for the german version of Windows 7 64 bit.
(CodeXL_Win_2.0.12406 installer)

**Comment by ilangal-amd** [comment]
Thanks for this report.
Can you help provide the installer log file to investigate this issue?

**Comment by klausklaus** [comment]
[MSI33392.TXT](https://github.com/GPUOpen-Tools/CodeXL/files/227581/MSI33392.TXT)

Here is the .LOG file. (Renamed to .txt for githup)

**Comment by nnorm** [comment]
Same issues here on Windows 10 Pro. 64 bit (French) (CodeXL_Win_2.0.12406.exe). Just before quitting the installer I found the main binary file (CodeXL.exe) and the program seems to works fine. If explained how to I can provide a log file of the installation. I tried MightTower solution but it didn't worked (maybe I have done it wrong?). Hope you find the solution quickly. :)

**Comment by elect86** [comment]
I had the same, it's because we are not using an english Win

Follow [this](http://windows.microsoft.com/en-us/windows/user-groups#1TC=windows-7) and add an `Users` group with your user in it

**Comment by nnorm** [comment]
Thanks! It worked. It didn't the first time because I was missing the "s" at the end of the group name. Hope this issue will soon be fixed. Thanks again! :)

**Comment by ilangal-amd** [comment]
Thanks for the reports and sorry for the issue. The log helped to find the root cause, I will update when a solution will be ready.

**Comment by miskeletor** [comment]
I had the same issue on Swedish Windows 10 Home.
The solution suggested by elect86 doesn't work on home edition but i found another one.

Start a CMD as administrator, then create the group and add your user:
```
net localgroup Users /add
net localgroup Users <user_name> /add
```

**Comment by sergioafeo** [comment]
Renaming whatever the localized version of the users group to "Users" allows the installation to continue and succeed. Concretely, for the german version of windows, renaming the "Benutzer" user group to "Users" does the trick. Should work similarly for all other localizations.

Most likely the group name is hardcoded in the installer, this should be done in windows through the SID: S-1-5-32-545. See here: https://support.microsoft.com/en-us/kb/243330

**Comment by julihocc** [comment]
Same issues here on Windows 10. 64 bit (Spanish) (CodeXL_Win_2.0.12406.exe).

**Comment by ilangal-amd** [comment]
Issue was fixed and released in version [2.1 windows installer](https://github.com/GPUOpen-Tools/CodeXL/releases/download/v2.1/CodeXL_Win_2.1.223.exe).

Ilan.
